modules/airgap/setup/get_artifacts.sh

```diff
@@ -4,14 +4,14 @@
 # set -x
 # echo "$@"
 
-# Usage: ./get_artifacts.sh k3s v1.27.5+k3s1
-# Usage: ./get_artifacts.sh rke2 v1.27.5+rke2r1 amd64 "flags" tar.gz
+# Usage: ./get_artifacts.sh "k3s" "v1.31.0+k3s1"
+# Usage: ./get_artifacts.sh "rke2" "v1.31.0+rke2r1" "amd64" "flags" "tar.gz"
```

Comment:
can we add basic validations here?
for mandatory vars that cannot get to here empty, like prod and version

Reply:
Updated

Reply:
not seeing